galaxy_importer/collection.py

```diff
@@ -175,4 +178,4 @@ def _extract_archive(fileobj, extract_dir):
                 )
                 if not link_target.startswith(os.path.abspath(extract_dir)):
                     raise exc.ImporterError("Invalid link target detected.")
-        tf.extractall(extract_dir)
+        tf.extractall(extract_dir, **kwargs)
```

Comment:
Completely unrelated, but when I switched the `test_eda` tests to be able to contribute to coverage and added capsys, I noticed the warning from tarfile that python 3.14 would change the behavior, and figured I could just address this quickly now.

Reply:
So, in this case, can a comment be added explaining it?

or maybe another conditional made instead of the `hasattr` e.g:

```py
_kwargs = {}
if sys.version_info >= (3, 12):
    _kwargs["filter"] = "data"
tf.extractall(extract_dir, **_kwargs)
```

So it becomes clear the reason for the conditional arg.

Reply:
Yeah, I'll add a comment.  The `filter` functionality has been backported within RHEL python, so a `hasattr` is the recommendation.

Reply:
Comment added, and variable renamed.